src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -103,10 +105,20 @@ export default class MxGraphRenderer {
             mxEdge.geometry.offset = new mxPoint(relativeLabelX, relativeLabelY);
           }
         }
+
+        this.insertMessage(edge, mxEdge);
       }
     });
   }
 
+  private insertMessage(edge: Edge, mxEdge: mxCell): void {
```

Comment:
:question: can we rename it into something different to better express what the function is doing,  like `insertMessageFlowIconIfNeeded`